cpp/include/cuml/fil/treelite_importer.hpp

```diff
@@ -377,11 +374,6 @@ struct treelite_importer {
         }
       }
     }
-    // Check base_scores
-    for (std::int32_t class_id = 1; class_id < tl_model.num_class[0]; ++class_id) {
-      ASSERT(tl_model.base_scores[0] == tl_model.base_scores[class_id],
-             "base_scores must be identical for all classes");
-    }
```

Comment:
Note. FIL can now handle vector bias (`base_score`), so this check is being removed.